pkgs/by-name/co/codd/generated.nix

```diff
@@ -0,0 +1,113 @@
+# This was generated with cabal2nix but contains important modifications like jailbreaking haxl
+# and disabling tests selectively, among a few other minor things.
+# Don't just blindly replace it with something generated by cabal2nix.
```

Comment:
I took a look but having an additional script and running instructions felt a bit overkill. Unless this is the standard for all packages now, I'd prefer to keep this as is, particularly since I'm the author of the tool and will be keeping this up-to-date myself.

One intermediate approach is to do some splitting without a bash script, though. Let me know what you prefer!